compiler/rustc_session/src/options.rs

```diff
@@ -1933,6 +1933,8 @@ written to standard error output)"),
         Requires `-Clto[=[fat,yes]]`"),
     wasi_exec_model: Option<WasiExecModel> = (None, parse_wasi_exec_model, [TRACKED],
         "whether to build a wasi command or reactor"),
+    wasm_c_abi: bool = (false, parse_bool, [TRACKED],
```

Comment:
can we make it an enum with two values instead?

Reply:
Done!